src/main/java/pl/bajerska/befindyourmeal/user/InvalidUserEmailException.java

```diff
@@ -0,0 +1,6 @@
+package pl.bajerska.befindyourmeal.user;
+
+public class InvalidUserEmailException extends Throwable {
```

Comment:
Good idea with creating own Exceptions :) However it would be better to inherit Exception, not Throwable https://www.javamex.com/tutorials/exceptions/exceptions_hierarchy.shtml